### PR TITLE
Format ws

### PR DESCRIPTION
**Pull request by OndraZizka**
Fix helloworld-html5 pom.xml and whitespace formatting for few other pom.xmls
Basically, it's now aligned to JBoss 4-spaces standard.
Also, I fixed 2 poms which made QS uncompilable.

### CODE REVIEW

**Comment by sgilda** [comment]
Hi Ondra, would you mind squashing the commits to make it easier to see which files were changed at a glance? :-)
Thanks, Sande

**Comment by OndraZizka** [comment]
Hi Sande, GitHub does it for you. Click on the Files tab - that shows the aggregation of changes.
I have changed all pom.xml's.

**Comment by OndraZizka** [comment]
BTW there's one orphaned directory, IIRC it's carmart - it's not listed as module.

**Comment by sgilda** [comment]
Thanks Onrda. I learn something every day! :-)

@rafabene , are you aware of an orphaned directory in carmart?

@pmuir: Is it still preferable to squash the commits before merging, or is it not important?

**Comment by pmuir** [comment]
@OndraZizka did you push the html5 changes to aerogear as well?

Yes, it would be good to squash this all down in to one commit

**Comment by OndraZizka** [comment]
No, changed just here.
Ok i will squash it.

**Comment by sgilda** [comment]
@OndraZizka : Just a reminder that I'm waiting for these commits to be squashed. :-)

**Comment by OndraZizka** [comment]
It took me like an hour to squash so I hope its worth it.

**Comment by OndraZizka** [comment]
Not really buildable after squash. - gwthelloworld and erraihelloworld

**Comment by OndraZizka** [comment]
Fixed

**Comment by OndraZizka** [comment]
Squashed again :)

**Comment by sgilda** [comment]
Thanks Ondra! I will try to review it today.

**Comment by sgilda** [comment]
Hi Ondra, 

There are 2 new pom.xml files in this commit: 

memcached-endpoint/pom.xml
rest-endpoint/pom.xml

However, there is no memcached-endpoint or rest-endpoint quickstart that I can see. Are these files supposed to be here?

**Comment by OndraZizka** [comment]
Could happen that I put it there during rebasing, while they were removed since my original base. I'll check that tomorrow.

**Comment by sgilda** [comment]
@OndraZizka : did you ever get a chance to check into the extra pom.xml files?

**Comment by OndraZizka** [comment]
Rebasing...

**Comment by OndraZizka** [comment]
Rebased, and the two pom.xml's removed:
  memcached-endpoint/pom.xml
  rest-endpoint/pom.xml

File: kitchensink-html5-mobile/pom.xml [hunk 1]
```diff
@@ -55,6 +55,7 @@
         <version.license.plugin>1.9.0</version.license.plugin>
         <version.surefire.plugin>2.4.3</version.surefire.plugin>
         <version.war.plugin>2.2</version.war.plugin>
+        <version.org.jboss.as.plugins.maven.plugin>7.2.Final</version.org.jboss.as.plugins.maven.plugin>
```
Comment (sgilda) [inline kitchensink-html5-mobile/pom.xml]: This quickstart originates here: https://github.com/aerogear/as-quickstarts/tree/master/kitchensink-html5-mobile; So this fix needs to be made in that repository. 

@OndraZizka : do you want to enter a JIRA for this?

**Comment by sgilda** [comment]
The helloworld-html5 and  kitchensink-html5-mobile quickstart originate here: https://github.com/aerogear/as-quickstarts/ so those fixes need to be made in that repository. 

Actually, there are quite a few properties defined in the  kitchensink-html5-mobile here that aren't in the Aerogear version. 

@OndraZizka : Do you want to enter a JIRA for this or can you make the same changes there?

**Comment by OndraZizka** [comment]
Reverted kitchensing changes and squashed.
Not sure about jira, when it's almost done? :8ball:

**Comment by sgilda** [comment]
@OndraZizka : I'm not clear what you mean by the comment "Not sure about jira, when it's almost done? :8ball: ". Does the kitchensink need that property defined?

**Comment by sgilda** [comment]
@pmuir: do you have any other comments? If not and you're OK with this, I can merge.

**Comment by OndraZizka** [comment]
:8ball:  is github's icon shortcut which got into comment by autocomplete...

**Comment by sgilda** [comment]
Merged.

**Pull request closed without merging**